packages/server/src/ui/routes/project-dashboard/graphs/category-score/category-score-graph.jsx

```diff
@@ -36,9 +36,17 @@ const VersionChangeWarning = () => {
   );
 };
 
-/** @param {{selectedBuildId: string|undefined, medianStatistics: Array<StatisticWithBuild>, versionChanges: Array<{build: LHCI.ServerCommand.Build}>, pinned: boolean}} props */
+const InfoTip = () => {
+  return (
+    <div className="category-score-graph__version-change-warning">
```

Comment:
mind creating a different class for this?

Reply:
darn, you got me. was hoping to cut corner here 😆